examples/rl/supply_chain/policy_trainer.py

```diff
@@ -3,39 +3,57 @@
 
 from functools import partial
 
-from maro.simulator.scenarios.supply_chain import ConsumerUnit, ManufactureUnit, ProductUnit
+from maro.simulator.scenarios.supply_chain import ConsumerUnit, ManufactureUnit, ProductUnit, SellerUnit
 from maro.simulator.scenarios.supply_chain.world import SupplyChainEntity
+from .algorithms.rule_based import or_policy_creator
 from .config import rl_algorithm, NUM_CONSUMER_ACTIONS
 from .env_helper import entity_dict
 from .state_template import STATE_DIM
 
 
-def get_policy_name(entity_: SupplyChainEntity) -> str:
-    if entity_.is_facility:
+def get_policy_name(entity: SupplyChainEntity) -> str:
+    if entity.skus is None:
```

Comment:
Is `entity.skus is None` a good condition to recognize facilities?

Reply:
My original base branch did not have ``is_facility`` attribute for ``SupplyChainEntity``. Changed now.